_SETUP_DEV_ENVIRONMENT.md

```diff
@@ -22,6 +22,64 @@ There are several ways to set up your development server and environment. Basica
 
 Note: to regenerate the docker configuration, use `php artisan sail:install --devcontainer`
 
+### Windows (natively)
+On Windows, probably the optimal option with respect to performance is
+to simply install a web server and a database server natively on the system,
+instead of using containers or virtual machines.
+However, the installation itself is a bit more complicated.
+(But actually, there aren't as many mysterious errors as with containers.)
+
+1. Download and install the following programs:
+    - [XAMPP](https://www.apachefriends.org/download.html) (the newest available one) – this includes the PHP interpreter, the Apache web server and the MySQL database server.
+        - Besides the obligatory options, we only need MySQL, phpmyadmin and Fake Sendmail.
+        - You don't need to allow it through the firewall, as we are only going to access these locally.
+    - [Composer](https://getcomposer.org/download/) – this is a package manager for PHP.
+        - It's enough to install it only for your account when asked.
+    - [NVM](https://github.com/coreybutler/nvm-windows/releases) (choose `nvm-setup.exe`) – this downloads Node.js and the NPM package manager, which will again download other things:D
+    - [Git](https://gitforwindows.org/) – the version manager.
+        - For the text editor, the simplest is to choose Windows' Notepad (or VS Code, if you have it). Otherwise, you can stick to the defaults.
+2. Open `C:\xampp\php\php.ini`. This is PHP's configuration file. We need to enable two extensions here for things to work.
+    - To do this, search for the lines `;extension=zip` and `;extension=gd`, and delete the semicolon (`;`) before them. (If there are more than one copies of them, it's enough to enable only one instance.)
+3. Open a command prompt (open Start menu and type `cmd`).
+4. Type `nvm install lts`. This command will install NPM for you.
+5. To be able to use it, you need to add the folder containing the NPM binary to the operating system's PATH variable (so that the command prompt will find it).
+    - Check the installed version of Node.js with `nvm list`
+    - Follow [these steps](https://www.mathworks.com/matlabcentral/answers/94933-how-do-i-edit-my-system-path-in-windows). Add the path `%USERPROFILE%\AppData\Roaming\nvm\vXX.XX.X`, where `XX.XX.X` is the version printed by `nvm list`.
```

Comment:
Avoid links to random pages which might not be available/updated later. Just mention adding it to the path, everyone can google it.